docs/build-process/schema-descriptor-build-process.md

```diff
@@ -0,0 +1,19 @@
+# Building a schema descriptor
+This page explains how to build a schema descriptor in a SpatialOS project that’s using the flexible project layout (FPL).
+
+## Schema descriptor introduction 
+The schema descriptor is a build artifact of a SpatialOS project [assembly](https://docs.improbable.io/reference/13.6/shared/glossary#assembly). It allows the SpatialOS Runtime to interpret your project’s schema and is necessary for starting a SpatialOS deployment both locally and in the cloud. You can generate a schema descriptor using the [schema compiler tool](https://docs.improbable.io/reference/latest/shared/schema/introduction#using-the-schema-compiler-directly).
+
+**Note:** In the [current project layout (CPL)](https://docs.improbable.io/reference/latest/shared/reference/project-structure), the spatial CLI automatically generates the schema descriptor. For an FPL project, you need to manually generate your schema descriptor.
```

Comment:
Sorry, I forgot the FPL docs would be on GitHub which means you can't use the fancy Improbadoc Tip/Info etc. macros :( We're moving these docs into the main docs at some point, right?

Reply:
Yes, the idea is to have the docs here first and then move them to the website when we "properly" release the feature. The docs in this repo are like a preview.